Yosemite/Yosemite/PointOfSale/Items/PointOfSaleItemService.swift

```diff
@@ -91,9 +91,9 @@ public final class PointOfSaleItemService: PointOfSaleItemServiceProtocol {
                                                            productImageSource: thumbnailSource,
                                                            productID: product.productID,
                                                            price: product.price,
-                                                           manageStock: false,
-                                                           stockQuantity: nil,
-                                                           stockStatusKey: ""))
+                                                           manageStock: product.manageStock,
+                                                           stockQuantity: product.stockQuantity,
+                                                           stockStatusKey: product.stockStatusKey))
```

Comment:
non-blocking thought: how about grouping the stock properties into a struct? Then, `StockFormatter.stockStatusLabel` just takes in the stock enum, and `productStockStatus: ProductStockStatus` can just be a private extension of the stock enum.

Reply:
That's a good point and sounds good, as implies changing the model and regenerate them I'll take a look into it separately.